reference/tools/tiup/mirrors.md

```diff
@@ -0,0 +1,181 @@
+  tiup mirrors local-path --arch amd64,arm --os linux,darwin    # Specify the architectures and OSs
+  tiup mirrors local-path --full                                # Build a full local mirrors
+  tiup mirrors local-path --tikv v4                             # Specify the version via prefix
+  tiup mirrors local-path --tidb all --pd all                   # Download all version for specific component
+Flags:
+      --overwrite                   Overwrite the exists tarball
+  -f, --full                        Build a full mirrors repository
+  -a, --arch strings                Specify the downloading architecture (default [amd64])
+  -o, --os strings                  Specify the downloading os (default [linux,darwin])
+      --tidb strings                Specify the versions for component tidb
+      --tikv strings                Specify the versions for component tikv
+      --pd strings                  Specify the versions for component pd
+      --playground strings          Specify the versions for component playground
+      --client strings              Specify the versions for component client
+      --prometheus strings          Specify the versions for component prometheus
+      --package strings             Specify the versions for component package
+      --grafana strings             Specify the versions for component grafana
+      --alertmanager strings        Specify the versions for component alertmanager
+      --blackbox_exporter strings   Specify the versions for component blackbox_exporter
+      --node_exporter strings       Specify the versions for component node_exporter
+      --pushgateway strings         Specify the versions for component pushgateway
+      --tiflash strings             Specify the versions for component tiflash
+      --drainer strings             Specify the versions for component drainer
+      --pump strings                Specify the versions for component pump
+      --cluster strings             Specify the versions for component cluster
+      --mirrors strings             Specify the versions for component mirrors
+      --bench strings               Specify the versions for component bench
+      --insight strings             Specify the versions for component insight
+      --doc strings                 Specify the versions for component doc
+      --ctl strings                 Specify the versions for component ctl
+  -h, --help                        help for tiup
+```
+
+The basic usage of the `tiup mirrors` command is as follows:
+
+{{< copyable "shell-regular" >}}
+
+```bash
+tiup mirrors <target-dir> [global-version] [flags]
+```
+
+- `target-dir`: used to specifies the directory in which cloned data is stored.
+- `global-version`: used to quickly set a global version for all components.
+
+The `tiup mirrors` command provides many optional flags (might provide more in the future). These flags can be divided into the following four categories according to their intended usages:
```

Comment:
```suggestion
The `tiup mirrors` command provides many optional flags (might provide more in the future). These flags can be divided into the following categories according to their intended usages:
```
It's better not to specify the number here, in case someone adds a new category below and forgets to change this number.